src/tir/usmp/transform/assign_pool_info.cc

```diff
@@ -48,18 +48,26 @@ class PoolInfoAssigner : public StmtExprMutator {
     ICHECK(target_host) << "main function does not have a target attr";
     WorkspaceMemoryPools workspace_pools =
         module->GetAttr<WorkspaceMemoryPools>(tvm::attr::kWorkspaceMemoryPools)
-            .value_or(WorkspaceMemoryPools({CreateDefaultMemoryPool(module)}));
+            .value_or(WorkspaceMemoryPools(
```

Comment:
Please create a new structure for ConstantMemoryPools (I mentioned this in another comment)

Reply:
ConstantPoolInfo should not be a part of WorskpaceMemoryPools